scripts/abins/spowdersemiempiricalcalculator.py

```diff
@@ -507,14 +506,9 @@ def calculate_isotropic_dw(self, *, q2: np.ndarray) -> np.ndarray:
         return iso_dw
 
     @staticmethod
-    def _isotropic_dw(*, frequencies, q2, a_trace, temperature):
+    def _isotropic_dw(*, q2, a_trace):
         """Compute Debye-Waller factor in isotropic approximation"""
-        if temperature < np.finfo(type(temperature)).eps:
-            coth = 1.0
-        else:
-            coth = 1.0 / np.tanh(frequencies * CM1_2_HARTREE / (2.0 * temperature * K_2_HARTREE))
```

Comment:
where have these coth terms gone?

Reply:
They have gone into the `a_trace` values. PowderCalculator applies the thermal occupation when forming the `a_tensors` and the traces are taken as a method on PowderData

Reply:
Thank you. I knew they went somewhere just wasn't sure where